README.md

```diff
@@ -69,6 +68,8 @@ vars:
     amazon_ads_<default_source_table_name>_identifier: your_table_name 
 ```
 
+ 
+ 
```

Comment:
This package will ultimately have the ability for customers to add passthrough metrics. We will want to include a relevant section. See how we did it in [TikTok Ads](https://github.com/fivetran/dbt_tiktok_ads_source#passing-through-additional-metrics) for reference.

Reply:
A good rule of thumb is that any source table that has `_report` will have the passthrough metrics included.

Reply:
Done.